components/pd/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "pd"
```

Comment:
maybe `tikv_pd` or `pd_client` is better to avoid someone thinking that it is the PD.

Reply:
oh, seem it is better to rename to pd_client?
/cc @overvenus

Reply:
Good idea!